### PR TITLE
Implement file checking for critical files

### PR DESCRIPTION
**Pull request by shadow93**
Should make life easier for server operators and boss devs.

### CODE REVIEW

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 1]
```diff
@@ -59,6 +59,12 @@ Updated by Wliu, Chris, Lawd, and Carge after Powerlord quit FF2
 #define MONOCULUS "eyeball_boss"
 #define DISABLED_PERKS "toxic,noclip,uber,ammo,instant,jump,tinyplayer"
 
+#define OVER_9000 "saxton_hale/9000.wav"
```
Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: I'd prefer to keep these undefined for now

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 2]
```diff
@@ -59,6 +59,12 @@ Updated by Wliu, Chris, Lawd, and Carge after Powerlord quit FF2
...
+#define OVER_9000_PREFIX "sound/saxton_hale/9000.wav"
+
+// In rare cases, some server operators may need to undef this.
+#define FILECHECKS_ON
```
Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: What rare cases?

Comment (Deathreus) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: Wouldn't it be easier to tie this to a ConVar bool?

Comment (shadow93) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: Transit of Venus has a rare case where his SM install can't find the files despite them existing.

Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: I would say that's a problem with his server then and FF2 shouldn't guard against it.

**Comment by shadow93** [comment]
Can you have a look at the comments i left regarding getting config names?

**Pull request merged**